package.json

```diff
@@ -14,6 +14,7 @@
     "test": "lage test",
     "lint": "lage lint",
     "lage": "lage",
+    "ci": "yarn build && yarn test && yarn lint && yarn checkchange",
```

Comment:
nice - I think all you need to do this: `yarn lage build test lint && yarn checkchange`

Reply:
:boom: Went from 4 sec to 3 sec